tests/conftest.py

```diff
@@ -79,7 +79,9 @@ def app_config(app_config):
     from oarepo_vocabularies.resources.config import VocabulariesResourceConfig
     from oarepo_vocabularies.services.config import VocabulariesConfig
 
-    app_config["VOCABULARIES_SERVICE_CONFIG"] = VocabulariesConfig
```

Comment:
Should stay here as app_config["VOCABULARIES_SERVICE_CONFIG"] = VocabulariesConfig